helphours/logger.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+
+
+class Logger():
+
+    # The various logging levels to mask out log entries.
+    # e.g. Using LEVEL_WARNING will mask out all DEBUG and INFO messages.
+    LEVEL_DEBUG = 10
+    LEVEL_INFO = 20
+    LEVEL_WARNING = 30
+    LEVEL_ERROR = 40
+    LEVEL_FATAL = 50
+
+    DEBUG_PREFIX = "[Debug]"
+    INFO_PREFIX = "[ Info]"
+    WARNING_PREFIX = "[ WARN]"
+    ERROR_PREFIX = "[ERROR]"
+    FATAL_PREFIX = "[FATAL]"
+
+    def __init__(self, email_notifier, admin_emails, log_file, log_level=0):
+        '''
+            Creates a new Logger instance.
+            email_notifier -- An instance of the Notifier class to send emails to admins
+            admin_emails -- List of email addresses to send select error and fatal messages to
+            log_file -- The name of the log file to create or append to
+            log_level -- Logging level. Used to mask out lower severity messages.
+                         Should be one of the LEVEL_* class constants defined above.
+        '''
+        self.email = email_notifier
+        self.admin_emails = admin_emails
+        self.log_file = log_file
+        self.log_level = log_level
+
+    # Helper Methods
+    def format_message(severity_prefix, message):
+        now = datetime.now()
+        time_string = now.strftime("[%b %d, %Y][%H:%M:%S]")
+        return f"{time_string}{severity_prefix}: {message}\n"
```

Comment:
Will there be a space between the time_string and the [INFO], [ERROR] such? If not, we should probably add one.

Reply:
Up to you if you think we need one. I think it looks alright with the square brackets and no space:
```
[Aug 16, 2020][15:21:50][ERROR]: Example message
```